x-pack/filebeat/docs/inputs/input-benchmark.asciidoc

```diff
@@ -0,0 +1,93 @@
+[role="xpack"]
+
+:type: benchmark
+
+[id="{beatname_lc}-input-{type}"]
+=== Benchmark input
+
+++++
+<titleabbrev>Benchmark</titleabbrev>
+++++
+
+beta[]
+
+The Benchmark input generates generic events and sends them to the output.  This can be useful when you want to benchmark the difference between outputs or output settings.
```

Comment:
[Doc lint]
```suggestion
The Benchmark input generates generic events and sends them to the output. This can be useful when you want to benchmark the difference between outputs or output settings.
```